tests/test_htmIndex.py

```diff
@@ -163,18 +163,19 @@ def testValidateParallax(self):
                             with self.assertRaises(ValueError):
                                 config.validate()
 
-    def testIngestVersionSet(self):
-        """Test that newly ingested catalogs get the correct version number set."""
+    def testIngestSetsVersion(self):
+        """Test that newly ingested catalogs get the correct version number set.
+        """
         # Test with multiple files and standard config
         config = self.makeConfig(withRaDecErr=True, withMagErr=True, withPm=True, withPmErr=True)
         # don't use the default depth, to avoid taking the time to create thousands of file locks
         config.dataset_config.indexer.active.depth = self.depth
         IngestIndexedReferenceTask.parseAndRun(
-            args=[self.input_dir, "--output", self.outPath+"/output_multifile",
-                  self.skyCatalogFile, self.skyCatalogFile],
+            args=[self.input_dir, "--output", self.outPath+"/output_setsVersion",
```

Comment:
Take the opportunity to introduce spaces either side of the `+`?
Here and below.

Reply:
Done, but it made the lower line exceed 110 characters! ;-)